build.gradle

```diff
@@ -372,9 +385,38 @@ task createApiDocsRedirect {
   }
 }
 
+task publishDocs {
+  dependsOn 'closeAndReleaseRepository'
+  doLast {
```

Comment:
I don't feel pulling here is that necessary but it does not hurt either. Ideally we would need to clone a new tmp_gh-pages before each release.